src/common/metrics/hospitalizations.ts

```diff
@@ -67,44 +68,40 @@ export const HOSPITALIZATIONS_DISCLAIMER =
   ', a pandemic think tank, recommends that hospitals maintain enough ICU capacity to double the number of COVID patients hospitalized.';
 
 export function hospitalOccupancyStatusText(projection: Projection) {
-  const currentIcuUtilization = projection.currentIcuUtilization;
-  const currentCovidICUPatients = projection.currentCovidICUPatients;
-  const totalICUCapacity = projection.totalICUCapacity;
-  const nonCovidPatients = Math.floor(projection.nonCovidPatients);
+  const icu = projection.icuHeadroomInfo;
```

Comment:
This code didn't actually change much, but I created icuHeadroomInfo as a container for all of the data we were pulling out of the projection, including the new metadata about actuals that we use to tweak the copy below.